core/arch/arm/plat-imx/config/imx6sll.h

```diff
@@ -0,0 +1,92 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright 2017-2019 NXP
+ *
```

Comment:
minor: empty line

Reply:
Addressed in a6bca6b924766606e377fc187444f92ce92122dc